run_ejb_app_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Runs EJB App tests (ejb-core/rust/ejb-app).
```

Comment:
Why don't we call `run_ejb_app_tests.sh` from ~~`run_all_tests.sh`~~ `run_native_integration_tests.sh`?

Reply:
I think it's just separated tests, not native_its